config.default.json

```diff
@@ -8,7 +8,7 @@
           "wss":["auto"]},
  "check_subdomains": true,
  "log_level":"debug",
- "bind_hostname": true,
+ "bind_address": true,
```

Comment:
Do we need to change anything so old `config.json` files still work? (These are only supported with `wpt serve` and not `wpt run`, AFAIK.)